### ISSUE TITLE
Avo locale config is ignored when Rails locale is set to something else than :en 

### ISSUE THREAD

**Issue by adrienpoly** [avo-hq/avo#1022]
### Describe the bug

I am working on an app and with AVO for the Admin. 
The app has a default locale in Rails set to `:fr` and available locales are [:fr, :en].

```ruby 
# config/application.rb
# Permitted locales available for the application
config.i18n.available_locales = [:fr, :en]

# Set default locale
config.i18n.default_locale = :fr
```
In initializers/avo.rb I have

```ruby
# config/initializers/avo.rb
## == Localization ==
config.locale = :en
```

This is a fairly new rails App with almost nothing yet in it.

### Expected behavior

What ever locale I set in `application.rb` I expect this value to be overwritten in the admin if I force the locale in the initializer/avo.rb. 

Maybe I am not understanding correctly the intent of this setting but that was my expectation.

### Actual behavior

I have an error message
![Capture d’écran 2022-07-05 à 2 37 07 PM](https://user-images.githubusercontent.com/7847244/177579366-57d4256d-8510-463e-bdf2-ba93389a2269.jpg)

### Current solution

I could solve my problem by adding an `avo.fr.yml` file. But I feel like this is only a patch

### System configuration
**Avo version**: 2.9

**Rails version**: 7

**Ruby version**: 3.1

**License type** (Community or Pro): Community

#### Impact

 - [ ] High impact
 - [ ] Medium impact
 - [x] Low impact

I could find a solution by adding avo.fr.yml

#### Urgency

 - [ ] High urgency
 - [ ] Medium urgency
 - [x] low urgency

**Comment by adrianthedev** [comment]
Hey @adrienpoly.

Sorry, but that's not what the `locale` option is used for. It will still use the Rails settings, in your case `default_locale = :fr`. It's a bit less opinionated that way.
You can force a locale on Avo using the `?force_locale=en` param in your route.

I understand it's not the best experience when you first use it. It would be cool to have such a feature in the future.

I'm closing this for now. Feel free to reopen it.

**Comment by adrienpoly** [comment]
Ok fine. 

May I ask for a clarification of what is this setting used for ?

From reading the [doc](https://docs.avohq.io/2.0/localization.html#setting-the-locale) I kind of understood it was the expected behavior.

**Comment by adrianthedev** [comment]
Fair enough! It wasn't used anywhere. It's fixed now and should work properly. I updated the docs too.

https://docs.avohq.io/2.0/localization.html#setting-the-locale
